app/controllers/cart_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CartController < ApplicationController
+  def create
+    if Cart.last
+      cart = Cart.last
+    else
+      cart = Cart.create()
+     end
+    photo = Photo.find(params[:photo_id])
+    if cart.add(photo, photo.price)
```

Comment:
Fix the code indentation around here... Also create an empty line between blocks of code to improve readability